Assets/XML/Units/CIV4SpawnInfos.xml

```diff
@@ -1029,13 +1029,17 @@ Using both <iGlobalTurns> and <iTurns> will not yield predictable results:
 			<!--			<iMaxLatitude>20</iMaxLatitude>-->
 			<!--			<iMinLongitude>-60</iMinLongitude>-->
 			<!--			<iMaxLongitude>60</iMaxLongitude>-->
-			<FeatureTerrainTypes>
-				<FeatureType>FEATURE_SAVANNA</FeatureType>
-				<FeatureType>FEATURE_FOREST</FeatureType>
+			<FeatureTypes>
+				<FeatureType>
+					<FeatureType>FEATURE_SAVANNA</FeatureType>
+					<FeatureType>FEATURE_FOREST</FeatureType>
+				</FeatureType>
+			</FeatureTypes>
```

Comment:
I think you got these Features nested 1 too many times. 
Should be same as the terrain.
Like this:
			<FeatureTypes>
				<FeatureType>FEATURE_SAVANNA</FeatureType>
				<FeatureType>FEATURE_FOREST</FeatureType>
			</FeatureTypes>